iceoryx_hoofs/platform/linux/include/iceoryx_hoofs/platform/getopt.hpp

```diff
@@ -0,0 +1,22 @@
+// Copyright (c) 2020 by Robert Bosch GmbH. All rights reserved.
+// Copyright (c) 2021 by Apex.AI Inc. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+// SPDX-License-Identifier: Apache-2.0
+#ifndef IOX_HOOFS_LINUX_PLATFORM_GETOPT_HPP
+#define IOX_HOOFS_LINUX_PLATFORM_GETOPT_HPP
```

Comment:
same here and with all the other files from platform?

Reply:
See [here](https://github.com/eclipse-iceoryx/iceoryx/pull/790/files#r630935855)